src/GameLogic/InventoryStorage.cs

```diff
@@ -110,7 +110,7 @@ public override async ValueTask<bool> AddItemAsync(byte slot, Item item)
             var isEquippedItem = this.IsWearingSlot(slot);
             if (isEquippedItem && this.EquippedItemsChanged is { } eventHandler)
             {
-                await eventHandler(new ItemEventArgs(convertedItem ?? item)).ConfigureAwait(false);
+                await eventHandler(new ItemEventArgs(convertedItem ?? item, true)).ConfigureAwait(false);
```

Comment:
unequipped->equipped: notify that isEquipped is true (no changes to ItemGroup byte will be made)